vagrant/Vagrantfile

```diff
@@ -600,26 +622,42 @@ Vagrant.configure('2') do |config|
                               mkfs.lustre --failover 10.73.20.22@tcp --ost --backfstype=zfs --fsname=zfsmo --index=6 --mgsnode=10.73.20.11@tcp:10.73.20.12@tcp ost6/ost6
-                              mkfs.lustre --failover 10.73.20.22@tcp --ost --backfstype=zfs --fsname=zfsmo --index=7 --mgsnode=10.73.20.11@tcp:10.73.20.12@tcp ost7/ost7
-                              mkfs.lustre --failover 10.73.20.22@tcp --ost --backfstype=zfs --fsname=zfsmo --index=8 --mgsnode=10.73.20.11@tcp:10.73.20.12@tcp ost8/ost8
-                              mkfs.lustre --failover 10.73.20.22@tcp --ost --backfstype=zfs --fsname=zfsmo --index=9 --mgsnode=10.73.20.11@tcp:10.73.20.12@tcp ost9/ost9
+                         SHELL
+
+        oss.vm.provision 'create-zfs-fs2',
+                         type: 'shell',
+                         run: 'never',
+                         inline: <<-SHELL
+                              mkfs.lustre --servicenode 10.73.20.21@tcp:10.73.20.22@tcp --mgs --backfstype=zfs mgt2/mgt2
+                              mkfs.lustre --reformat --failover 10.73.20.22@tcp --mdt --backfstype=zfs --fsname=zfsmo2 --index=0 --mgsnode=10.73.20.21@tcp:10.73.20.22@tcp mdt2-0/mdt2-0
+                              mkfs.lustre --failover 10.73.20.22@tcp --ost --backfstype=zfs --fsname=zfsmo2 --index=0 --mgsnode=10.73.20.21@tcp:10.73.20.22@tcp ost2-0/ost2-0
+                              mkfs.lustre --failover 10.73.20.22@tcp --ost --backfstype=zfs --fsname=zfsmo2 --index=1 --mgsnode=10.73.20.21@tcp:10.73.20.22@tcp ost2-1/ost2-1
                          SHELL
 
          oss.vm.provision 'mount-zfs-fs',
                          type: 'shell',
                          run: 'never',
                          inline: <<-SHELL
-                              mkdir -p /lustre/zfsmo/ost{0..9}
+                              mkdir -p /lustre/zfsmo/ost{0..6}
                               mount -t lustre ost0/ost0 /lustre/zfsmo/ost0
                               mount -t lustre ost1/ost1 /lustre/zfsmo/ost1
                               mount -t lustre ost2/ost2 /lustre/zfsmo/ost2
                               mount -t lustre ost3/ost3 /lustre/zfsmo/ost3
                               mount -t lustre ost4/ost4 /lustre/zfsmo/ost4
                               mount -t lustre ost5/ost5 /lustre/zfsmo/ost5
                               mount -t lustre ost6/ost6 /lustre/zfsmo/ost6
-                              mount -t lustre ost7/ost7 /lustre/zfsmo/ost7
-                              mount -t lustre ost8/ost8 /lustre/zfsmo/ost8
-                              mount -t lustre ost9/ost9 /lustre/zfsmo/ost9
+                         SHELL
+
+          oss.vm.provision 'mount-zfs-fs2',
+                         type: 'shell',
+                         run: 'never',
+                         inline: <<-SHELL
+                              mkdir -p /lustre/zfsmo2/{mgt2,mdt2-0}
```

Comment:
Can we setup `mgt2` on `mds2` as main server and `mds1` as fallback (mirrored from how `mgt` is setup)? Would be more symmetrical.